### ISSUE TITLE
firebug reports syntax error

### ISSUE THREAD

**Issue by illori** [SimpleMachines/SMF#1817]
SyntaxError: syntax error

});

index.php (line 483, col 3)

```
        <script><!-- // --><![CDATA[
            var oDraftAutoSave = new smf_DraftAutoSave({
                sSelf: 'oDraftAutoSave',
                sLastNote: 'draft_lastautosave',
                sLastID: 'id_draft',
                sSceditorID: 'message',
                sType: 'post',
                iBoard: 1,
                iFreq: 
            });
```

line 483 is the }); line in the above.

**Comment by Arantor** [comment]
Full posting? Posting from quick reply? PM posting?

**Comment by illori** [comment]
full posting page, i did not check the others

**Comment by Arantor** [comment]
And no errors in the error log?

**Comment by illori** [comment]
i just have an error about the drafts_autosave_frequency would that be related?

**Comment by Arantor** [comment]
Might be :laughing:

**Comment by illori** [comment]
then lets see if oldies can fix that and i can test this after.

Type of error: Undefined
8: Undefined index: drafts_autosave_frequency
github2.1/index.php?action=post;topic=21.0;last_msg=70
File: F:/wamp/www/github2.1/Themes/default/GenericControls.template.php
Line: 181

**Comment by Arantor** [comment]
Fixing one should fix the other...

**Comment by illori** [comment]
i hope so... now i hope oldies thinks this is bug and not a blank field in admin panel somewhere... i dont see any setting related to drafts as blank.

**Comment by Arantor** [comment]
But that only works if you're using the full editor... what about autosaving from quick reply?

**Comment by illori** [comment]
I saved from the quick reply and then it sends you to the full reply page

**Comment by Arantor** [comment]
That wasn't what I meant. Oh well.

**Comment by Oldiesmann** [comment]
Fixed quick reply autosave issue in #1822...
